### PR TITLE
Update internal CLI documentation

### LINKED ISSUE #2810: Update the "internal" CLI Meilisearch documentation
The current "internal" documentation of Meilisearch (when doing `cargo run -- --help`) is not always accurate. We would like to make it up-to-date with the official Meilisearch documentation.

Using this [link](https://docs.meilisearch.com/learn/configuration/instance_options.html#all-instance-options), update the comments in the following lines to make them correspond to the documentation description.

https://github.com/meilisearch/meilisearch/blob/2827ff7957675c3deb261d582168d1ca4cf6fe11/meilisearch-http/src/option.rs#L23-L169

Once #2745 is merged, you should update the comments in it as well to stay in sync between the cli and the default config file.

### PR DESCRIPTION
**Pull request by jeertmans**
# Pull Request

## Related issue
Fixes #2810

## What does this PR do?
- Make internal CLI documentation match the online one

## Remarks
- Scope is limited to `meilisearch/meilisearch-http/src/option.rs`, not the flattened structs: should I also take care of them?
- Could not find online docs for `enable_metrics_route`
- Max column width wrapping was done by hand, so may not be perfect
- I removed the links from the internal doc: should I put them back?

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

File: meilisearch-http/src/option.rs [hunk 1]
```diff
@@ -210,7 +271,17 @@ pub struct Opt {
     #[serde(default = "default_dumps_dir")]
     pub dumps_dir: PathBuf,
 
-    /// Set the log level. # Possible values: [ERROR, WARN, INFO, DEBUG, TRACE]
+    /// Defines how much detail should be present in Meilisearch's logs.
+
+    /// Meilisearch currently supports four log levels, listed in order of increasing verbosity:
```
Comment (dichotommy) [inline meilisearch-http/src/option.rs:276]:
```suggestion
    /// Meilisearch currently supports five log levels, listed in order of increasing verbosity:
```

Comment (jeertmans) [reply meilisearch-http/src/option.rs:276]: Indeed, didn't even catch this!

**Comment by curquiza** [comment]
bors try

**Comment by jeertmans** [comment]
Oups, seems like I should have called rustfmt @curquiza. It is now done :-)

**Review by curquiza — APPROVED** [review]
Thanks for the PR @jeertmans 

I requested some changes after discussing with @dichotommy: we think only the basic description is enough. Sorry the issue was not clear enough

Also, I think you missed some options like `HTTP_ADDR`, if I'm correct.

💡 Of course, you can apply my suggestions of change directly on this PR

Thanks again for the PR

**Review by curquiza — CHANGES_REQUESTED** [review]
Oops sorry I made a mistake: I approved instead of request changes 😂

**Comment by jeertmans** [comment]
Yes, indeed, I missed some attributes @curquiza! I applied requested changes, and I added missing attributes.

I think structs `IndexerOpts` and `SchedulerConfig` also need to be updated, but maybe that should be subject to another PR? Otherwise, I can update the docs here too.

**Comment by jeertmans** [comment]
I'm fine with one PR @curquiza ;-)

I just added the 3 missing docs.

**Review by curquiza — APPROVED** [review]
Good for me! thanks a lot @jeertmans for being such reactive

bors merge

**Comment by jeertmans** [comment]
It was a pleasure @curquiza! Thank you for your reactivity on my PR(s), it's always nice for contributors to receive feedback!

**Pull request merged**